tests/fixture/testnet/local/README.md

```diff
@@ -0,0 +1,211 @@
+
+// Use URIs to interact with the network
+
+// Stop all nodes in the network
+network.Stop()
+```
+
+If non-default node behavior is required, the `LocalNetwork` instance
+supplied to `StartNetwork()` can be initialized with explicit node
+configuration and by supplying a nodeCount argument of `0`:
+
+```golang
+network, _ := local.StartNetwork(
+    ctx,
+    ginkgo.GinkgoWriter,
+    "",
+    &local.LocalNetwork{
+        LocalConfig: local.LocalConfig{
+            ExecPath: "/path/to/avalanchego",
+        },
+        Nodes: []*LocalNode{{                                // node1 configuration is customized
+            Flags: FlagsMap{                                 // Any and all node flags can be configured here
+                cfg.DataDirKey: "/custom/path/to/node/data",
+            },
+        },{                                                  // node2 uses default configuration
+        },{                                                  // node3 uses default configuration
+        },{                                                  // node4 uses default configuration
+        },{                                                  // node5 uses default configuration
+        }},
+	},
+	0,                                                       // Node count must be zero when setting node config
+    50,
+)
+```
+
+Further examples of use code-based usage are located in the [e2e
+tests](../../../e2e/e2e_test.go).
+
+## Networking configuration
+
+By default, nodes in a local network will be started with staking and
+API ports set to `0` to ensure that ports will be dynamically
+chosen. The testnet fixture discovers the ports used by a given node
+by reading the `[base-data-dir]/process.json` file written by
+avalanchego on node start. The use of dynamic ports supports testing
+with many local networks without having to manually select compatible
+port ranges.
+
+## Configuration on disk
+
+A local network relies on configuration written to disk in the following structure:
+
+```
+HOME
+└── .testnetctl                                          // Root path for tool
+    └── networks                                         // Default parent directory for local networks
+        └── 1000                                         // Network path with form [network-id] starting from 1000
+            ├── NodeID-37E8UK3x2YFsHE3RdALmfWcppcZ1eTuj9 // The ID of a node is the name of its data dir
+            │   ├── chainData
+            │   │   └── ...
+            │   ├── config.json                          // Node flags
+            │   ├── db
+            │   │   └── ...
+            │   ├── logs
+            │   │   └── ...
+            │   ├── plugins
+            │   │   └── ...
+            │   └── process.json                         // Node process details (PID, API URI, staking address)
+            ├── chains
+            │   └── C
+            │       └── cchain_config.json               // C-Chain config for all nodes
```

Comment:
Why is the file name `cchain_config.json` when it is already in the `C` dir? I feel like this should just be `config.json`